sdk/textanalytics/ai-text-analytics/test/textAnalyticsClient.spec.ts

```diff
@@ -139,6 +139,8 @@ describe("[AAD] TextAnalyticsClient", function() {
           assert.isAtLeast(aspect.confidenceScores.positive, 0);
           assert.equal(aspect.confidenceScores.neutral, 0);
           assert.isAtLeast(aspect.confidenceScores.negative, 0);
+          assert.equal(aspect.offset, 32);
```

Comment:
It shouldn't affect any of these tests yet, but keep in mind that when the encoding feature lands, offsets might change with that parameter.